make.go

```diff
@@ -236,12 +236,16 @@ func createGitRepository(debsrc, gopkg, orig string) (string, error) {
 		return dir, err
 	}
 
-	if err := runGitCommandIn(dir, "config", "user.name", getDebianName()); err != nil {
-		return dir, err
+	if debianName := getDebianName(); debianName == "TODO" {
```

Comment:
Here and below, the condition is reversed, it needs to be `debianName != "TODO"`